openedx/features/edly/utils.py

```diff
@@ -188,6 +190,10 @@ def set_global_course_creator_status(request, user, set_global_creator):
     from course_creators.models import CourseCreator
 
     request_user = request.user
+    is_edly_panel_admin_user = request_user.groups.filter(name=settings.EDLY_PANEL_ADMIN_USERS_GROUP).exists()
```

Comment:
`EDLY_PANEL_ADMIN_USERS_GROUP ` is defined in edly_panel_app.api.v1.constants.. previous it was in settings.. kindly use the one's defined in edly_panel_app.

Reply:
This is not Edly panel app. this is features Edly app.